src/main/java/dev/socketmods/socketnukes/utils/HolidayTracker.java

```diff
@@ -0,0 +1,59 @@
+package dev.socketmods.socketnukes.utils;
+
+import java.time.LocalDate;
+
+public class HolidayTracker {
+
+    private static final boolean ENABLE_HATS = isChristmas() || isAprilFools();
+
+    /**
+     * Whether to force enable the Vanity Hats <br>
+     * We currently only enable the Hats for Christmas or April Fools
+     */
+    public static boolean enabledHats() {
+        return ENABLE_HATS;
+    }
+
+    //------------------------------------------------------------------------------------------------------------------
+
+    /**
+     * @return true if the date is between to March 31 - April 2
+     */
+    private static boolean isAprilFools() {
+        LocalDate now = LocalDate.now();
+
+        LocalDate march30 = now.withMonth(MARCH).withDayOfMonth(30);
+        LocalDate april3 = now.withMonth(APRIL).withDayOfMonth(3);
+
+        return now.isAfter(march30) && now.isBefore(april3);
+    }
+
+    /**
+     * @return true if the date is between to December 23 - January 2
+     */
+    private static boolean isChristmas() {
+        LocalDate now = LocalDate.now();
+
+        LocalDate december23 = now.withMonth(DECEMBER).withDayOfMonth(23);
+        LocalDate january3 = now.withMonth(JANUARY).withDayOfMonth(3);
+
+        return now.isAfter(december23) || now.isBefore(january3);
+    }
```

Comment:
How safe is this across calendars? Is there a possibility of this causing an error due to a user's calendar/timezone settings? 
We may want to consider having these surrounded in a `try`-`catch` block, to make sure that under no circumstances will this code cause a crash in the future.

Reply:
LocalDate uses the `ISO8601` Calendar System so this should be safe, you did remind me that `LocalDate#now` may actualy throw if the System Default Zone is invalid, I'll create a wrapper function to handle that case.